### ISSUE TITLE
Tests succeed no matter what

### ISSUE THREAD

**Issue by solongordon** [janestreet/ppx_expect#1]
I'm trying out ppx_expect, and try as I might I can't get a test to fail. Would you please let me know if I'm missing something?
```bash
cat > foo.ml <<EOF
open Core.Std

let%expect_test "addition" =
  printf "%d" (1 + 2);
  [%expect {| 4 |}]
EOF

ocamlfind ocamlc -linkpkg -thread \
  -package core,ppx_expect \
  -ppxopt 'ppx_expect,-inline-test-lib foo' \
  -o foo.out foo.ml

./foo.out inline-test-runner foo -verbose
```
Output:
```
File "foo.ml", line 3, characters 0-73: addition (0.001 sec)
```

**Comment by diml** [comment]
Indeed, I can reproduce this behavior. I'll need to debug this

**Comment by diml** [comment]
I found the problem, one part of ppx\_expect is missing, so it's basically not usable at the moment. Sorry for that, we'll fix the package. Since it's not a small change, we'll do it next week

**Comment by diml** [comment]
This is now fixed in 113.33.03, there is a working example there: https://github.com/janestreet/jane-street-tests/tree/master/test-inline-tests/with-oasis-method2. The README has been updated as well
